static/app/views/issueList/groupSearchViewTabs/editableTabTitle.tsx

```diff
@@ -94,17 +95,22 @@ function EditableTabTitle({
             value={inputValue}
             onChange={handleOnChange}
             onKeyDown={handleOnKeyDown}
-            onDoubleClick={() => isSelected && setIsEditing(true)}
+            onDoubleClick={() => setIsEditing(true)}
             onBlur={handleOnBlur}
             ref={inputRef}
             style={memoizedStyles}
             isEditing={isEditing}
-            onFocus={e => e.target.select()}
             onPointerDown={e => {
               e.stopPropagation();
+              if (!isEditing) {
+                e.preventDefault();
+              }
             }}
             onMouseDown={e => {
               e.stopPropagation();
+              if (!isEditing) {
+                e.preventDefault();
+              }
```

Comment:
This is very weird - the default behavior that this is preventing is clicking between characters in the tab title. When editing is active, this behavior is expected since the user should be able to click between characters. When editing is not active, preventing this behavior prevents the tab from being selected on a single click.